

&lt;H1&gt;

ProteoIDViewer 

&lt;/H1&gt;

 An open source viewer for mzIdentML



&lt;H2&gt;

ProteoIDViewer 1.4.1 Download: https://mzidentml-viewer.googlecode.com/svn/ProteoIDViewer-1.4.1.zip 

&lt;/H2&gt;

 This version supports the developing mzIdentML 1.2 (with full support for protein inference results) as well as the stable mzIdentML 1.1 release.



&lt;H2&gt;

Note about Downloads

&lt;/H2&gt;


Releases older than ProteoIDViewer 1.3 are hosted here under Downloads: https://code.google.com/p/mzidentml-viewer/downloads/list. Since Google code is retiring the Downloads section of these repositories, a new solution will soon be adopted for hosting our downloadable versions long term.






**Citation**
<br>If you use this resource in your work, please cite:<br>
<br>
<BR><br>
<br>
<br>
Ghali, F., Krishna, R., Lukasse, P., Martinez-Bartolome, S., Reisinger, F., Hermjakob, H., Vizcaino, J. A., and Jones, A. R. (2013) A toolkit for the mzIdentML standard: the ProteoIDViewer, the mzidLibrary and the mzidValidator. Molecular & Cellular Proteomics, mcp.O113.029777. <a href='http://www.mcponline.org/content/early/2013/06/28/mcp.O113.029777.abstract'>http://www.mcponline.org/content/early/2013/06/28/mcp.O113.029777.abstract</a>

<br>
<b>Overview</b>
<p>The Proteomics Standards Initiative released the mzIdentML standard format for peptide and protein identification, which can represent the output of most used search engines, such as Mascot, The Open Mass Spectrometry Search Algorithm (OMSSA) , and X!Tandem. The mzIdentML standard represents a peptide-spectrum match (PSM), in addition to a reference to a peptide sequence and an identifier for the matching spectrum presented in an external file. This PSM is accompanied by a score, rank, and the collection of all protein sequences that include the corresponding peptide. The mzIdentML file contains all the possible protein sequences that have been identified (<a href='http://www.psidev.info/mzidentml'>www.psidev.info/mzidentml</a>).</p><p>In order to allow the experimental proteomics community to use the mzIdentML standard, we developed a Graphical User Interface, <em>mzIdentML Viewer</em>, an open source viewer, to simplify proteomics data sharing for laboratory scientists to solve one of the main difficulties to open proteomic data analysis and large scale data sharing (<a href='http://code.google.com/p/mzidentml-viewer/'><a href='http://code.google.com/p/mzidentml-viewer/'>http://code.google.com/p/mzidentml-viewer/</a></a>).</p><p>The laboratory scientists are the main users of the mzIdentML Viewer which does not require bioinformatics support to get started. The viewer can be simply downloaded and installed with no complex setup procedures. Crucially, mzIdentML Viewer aims of providing intuitive and useful views of peptides/proteins and the search metadata.&nbsp; Moreover, the software calls any of the functions within mzIdentML Library, and exporters to spreadsheets.</p><p>The viewer provides several different methods for visualising data:</p><ul><li>Protein-view: all proteins identified and the supporting peptide evidence for each identification</li><li>Spectrum-view: ranked identifications for each input spectrum and peptide fragmentation data.</li><li>Peptide-view: all peptides identified, and potential protein mappings for each peptide</li><li>Global-view: global statistics include total peptide-spectrum matches, FDR, in addition to plotting FDR data into graphs for better visualisation.</li><li>Protocols-view: spectrum identification protocol and protein detection protocol data</li></ul><p></p><p>Viewer based on the MzIdentML API Google code:</p><p>&nbsp;</p><p><a href='http://code.google.com/p/mzidentml-viewer/'>http://code.google.com/p/mzidentml-viewer/</a></p>
<img src='http://mzidentml-viewer.googlecode.com/files/Picture1.png'>
<br>
<br>
<BR><br>
<br>
<br>
<img src='http://mzidentml-viewer.googlecode.com/files/Picture2.png'>
<br>
<br>
<BR><br>
<br>
<br>
<img src='http://mzidentml-viewer.googlecode.com/files/Picture3.png'>